### PR TITLE
Update README.md

### LINKED ISSUE #42: Can it process an ordered adapter on response type?
For instance, when my json looks like:
```
{
   "data" : [
      {
         "book_id" : "doe",
         "author_data" : [
            {
               "name" : "John Doe",
               "id" : "john_doe"
            }
         ],
         "title" : "Doe The John Doe",
      }
   ],
   "index_searched" : "book_id"
}
```
Could it be annotated like this?
```
@GET("/book/{key}")
  @Wrapped("data") @FirstElement fun aBook(@Path("key") key: String): Observable<Book>
```

### PR DESCRIPTION
**Pull request by serj-lotutovici**
Closes #42 and #39

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,26 +9,59 @@ Moshi Lazy Adapters
...
----
+##### How To Use It
 
-Moshi-Lazy-Adapters is **not** forcing any of it's own adapters by default. To leverage from any of
-the provided annotations/adapters add their respective factory to your `Moshi.Builder`:
+This library is **not** forcing any of it's own adapters by default. To leverage from any of
+ the provided annotations/adapters add their respective factory to your `Moshi.Builder`:
 
-```java
+```
```
Comment (vanniktech) [inline README.md]: missing `java`

File: README.md [hunk 2]
```diff
@@ -9,26 +9,59 @@ Moshi Lazy Adapters
...
+// Creates a Moshi instance with an adapter that will handle the @Wrapped annotations
```
Comment (vanniktech) [inline README.md]: nit: could be a sentence

File: README.md [hunk 3]
```diff
@@ -9,26 +9,59 @@ Moshi Lazy Adapters
...
+  @GET("/data")
+  @Wrapped(path={"result", "data"}) @FirstElement MyData getData();
+}
+
+```
+
+The resulting `JsonAdapter` will unwrap a list of `MyData` from the json response and return only 
+ the first element of that list.
+ 
+One **important** concept to keep in mind, is that the order of declared annotations in the example
```
Comment (vanniktech) [inline README.md]: `,` after `is`?

**Comment by serj-lotutovici** [comment]
@vanniktech updated to moshi 1.5 in this pr #57.

**Pull request merged**